com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ModelBuilder.java

```diff
@@ -35,20 +34,20 @@
 import com.dynamo.graphics.proto.Graphics.VertexAttribute;
 import com.dynamo.render.proto.Material.MaterialDesc;
 import com.dynamo.rig.proto.Rig.RigScene;
-import com.google.protobuf.TextFormat;
-
 
+// for editing we use ModelDesc but in runtime Model
+// make sure '.model' and '.modelc' corresponds to right classes
+@ProtoParams(srcClass = ModelDesc.class, messageClass = Model.class)
 @BuilderParams(name="Model", inExts=".model", outExt=".modelc")
-public class ModelBuilder extends Builder<Void> {
+public class ModelBuilder extends ProtoBuilder<ModelDesc.Builder> {
```

Comment:
This is one of a very few places where we use different input and output formats. 
In the future, I hope we will fix transform function and make this transformation point from one message to another clear